tools/prepare_dataset.py

```diff
@@ -75,9 +75,9 @@ def parse_args():
 
     print("List file {} generated...".format(args.target))
 
-    subprocess.check_call(["python",
+    subprocess.check_call(["python3",
```

Comment:
python3 may not be always available

Reply:
python2 is obsolete

Reply:
Let user decide which version to use, python is a link to whatever version.